_drafts/rse-at-egu.md

```diff
@@ -0,0 +1,73 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After the short talks, we took advantage of the group size, had a complete round of short introductions, and enganged in a discussion.
+We found a big challenge was reaching people engaged in RSEng activities at their work, especially outside of ESSI, and those for whom the "RSE" label does not ring a bell yet.
+The need for outreach also applies to holders of an office within the union, to reach better acknowledgements of the needs of RSEs in universities, scientific unions, and at scientific conferences.
+
+Some good ideas came up and we plan to reach out to EGU and ESSI leadership and share them.
+For example, EGU could increase recognition of RSEs and their work by awarding a medal, by offering a special poster track (not falling under submission limitations) for contributions to scientific software, or by tagging abstracts as "RSE" similarly to the "ECS" badges.
+These are "top-down" ideas that we would like towards within the organisation.
+The potential and usefulness of such domain-specific (taking all EGU members as a group for a moment) actions, who are lateral to the national chapters, was commonly agreed on.
+But there were also "bottom-up" actions who promise to support the RSE organisation's causes, such as scientists suggesting sessions related to RSE roles and activities within their divisions, offering short courses specifically around RSEng capabilities (and also labeling the numerous existing courses as such), or organising software and [data carpentry courses](https://carpentries.org/) in the week before or after the conference.
+If you think about convening a session on scientific software, please [get in touch](mailto:daniel.nuest@uni-muenster.de)!
+Who would not want to join _"Software development for professors - what does my student do with his computer"_?
+Jokes apart, it was great to follow the lively discussion, which was enriched with a common understanding of the values and importance of diversity and openness.
+
+An important aspect of structured RSE activities are surveys, because the people self-identifying as RSE are diverse and wide-spread and the roles that RSEs play in scientific research are manifold.
+We want to contribute to the process of understanding the needs and diversity of people involved in research software engineering with the following survey on EGU attendees:
```

Comment:
Unify capitalization of "Research Software Engineering"/"res soft eng" across whole text